x-pack/plugins/apm/public/components/shared/Stacktrace/__test__/Stackframe.test.tsx

```diff
@@ -16,7 +17,7 @@ describe('Stackframe', () => {
     let wrapper: ReactWrapper;
     beforeEach(() => {
       const stackframe = stacktracesMock[0];
-      wrapper = mount(<Stackframe stackframe={stackframe} />);
+      wrapper = mountWithIntl(<Stackframe stackframe={stackframe} />);
```

Comment:
`Stackframe` contains `FormattedMessage` as a child. `FormattedMessage` is required `intl` in the context. That's why we need to mount `Stackframe` with `intl` object in the context as `mountWithIntl` does it.
 
Otherwise the error will be in console during tests running `[React Intl] Could not find required 'intl' object. <IntlProvider> needs to exist in the component ancestry. Using default message as fallback.`